stylus/components/button.styl

```diff
@@ -0,0 +1,30 @@
+// Buttons of any kind
+.coz-btn
+    box-sizing border-box
+    display inline-block
+    vertical-align top
+    margin 0 em(8px)
+    border 1px solid grey-09
+    height: em(43px)
```

Comment:
I completely agree about no-colons it's much cleaner using stylus, and same thing about aligned values (even if less important to my mind).